src/main/java/org/hobbit/core/components/AbstractTaskGenerator.java

```diff
@@ -132,6 +132,7 @@ public void init() throws Exception {
         dataGenReceiver = DataReceiverImpl.builder().dataHandler(new DataHandler() {
             @Override
             public void handleData(byte[] data) {
+            	
```

Comment:
If that is the only change in this class, it does not make sense to me to have it. Please try to avoid changes like that and revert the change here.